tests/dataframe_tests/test_dataframe_injection.py

```diff
@@ -99,6 +100,17 @@ def test_get_leaves(translator):
             [f"file__0_Translator.collect_collecting_child__collected_collecting_dfs__{n}"]
         )
         input_collected_collecting_dfs[n] = input_df
+    schema = StructType([StructField('_file_id', StringType(), True), StructField('project_code', StringType(), True),
+                StructField('project_name', StringType(), True), StructField('program_name', StringType(), True),
+                StructField('project_id', StringType(), True), StructField('cmc_data_type', StringType(), True),
+                StructField('cmc_title', StringType(), True), StructField('cmc_subject', StringType(), True),
+                StructField('cmc_source', StringType(), True), StructField('cmc_relation', StringType(), True),
+                StructField('object_id', StringType(), True), StructField('md5sum', StringType(), True),
+                StructField('file_name', StringType(), True), StructField('file_size', LongType(), True),
+                StructField('data_format', StringType(), True), StructField('data_type', StringType(), True),
+                StructField('state', StringType(), True), StructField('data_category', StringType(), True),
+                StructField('source_node', StringType(), True)]
+    )
```

Comment:
what is this for?

Reply:
that's the scheme strictly matches with the dataset that we have in the test. I intended to use that to load the data produced by the ETL process in QA environment. But it failed to load because the LongType in schema
check it here @paulineribeyre 
https://github.com/uc-cdis/tube/actions/runs/5049205128/jobs/9058416638#step:24:493

Reply:
so you were going to add it to the `get_dataframes_from_names` call on line 114, right? right now `schemas` is not used anywhere

Reply:
yes, I commented it out